packages/i18n/src/write/properties/create.ts

```diff
@@ -22,11 +22,47 @@ export async function createPropertiesI18nEntries(
     fs?: Editor
 ): Promise<boolean> {
     if ((!fs && !(await doesExist(i18nFilePath))) || (fs && !fs.exists(i18nFilePath))) {
-        let content = '# Resource bundle \n';
-        if (root) {
-            content = `# This is the resource bundle for ${basename(root)}\n`;
-        }
-        await writeFile(i18nFilePath, content, fs);
+        await createNewI18nFile(i18nFilePath, root, fs);
     }
     return await writeToExistingI18nPropertiesFile(i18nFilePath, newI18nEntries, fs);
 }
+
+/**
+ * Creates or overwrites i18n entries in the `i18n.properties` file.
+ *
+ * @param {string} i18nFilePath - Absolute path to the `i18n.properties` file.
+ * @param {NewI18nEntry[]} newI18nEntries - New i18n entries to be added or updated.
+ * @param {string []} keysToRemove - Keys to be removed from the properties file.
+ * @param {string} root -Project root, optionally used in the comment if the file is newly generated.
+ * @param {Editor} fs - Optional `mem-fs-editor` instance. If provided, its API is used instead of Node's `fs`.
+ * @returns Promise that resolves when the operation is complete.
+ * @description If `i18n.properties` file does not exits, it tries to create.
+ */
+export async function createOrReplaceI18nEntries(
```

Comment:
`replace` is a bit misleading as it implies that something will be added at the same place, which is not the case here. Maybe something similar to `removeAndCreateI18nEntries` would be better fit.